lustre_exporter_test.go

```diff
@@ -137,1159 +186,1167 @@ func blacklisted(blacklist []string, metricName string) bool {
-		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "7"}}, 1.0469836e+07, false},
-		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "8"}}, 5.745458e+06, false},
-		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "9"}}, 517490, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "connect"}, {"ost", "lustrefs-OST0001"}}, 2, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "create"}, {"ost", "lustrefs-OST0001"}}, 3, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "get_info"}, {"ost", "lustrefs-OST0001"}}, 1, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "ping"}, {"ost", "lustrefs-OST0001"}}, 14630, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "set_info_async"}, {"ost", "lustrefs-OST0001"}}, 2, false},
-		{"lustre_stats_total", "Number of operations the filesystem has performed.", counter, []labelPair{{"operation", "statfs"}, {"ost", "lustrefs-OST0001"}}, 314187, false},
-		{"lustre_grant_precreate_capacity_bytes", "Maximum space in bytes that clients can preallocate for objects", gauge, []labelPair{{"ost", "lustrefs-OST0001"}}, 1.0174464e+07, false},
-		{"lustre_read_samples_total", "Total number of reads that have been recorded.", counter, []labelPair{{"ost", "lustrefs-OST0001"}}, 1, false},
-		{"lustre_blocksize_bytes", "Filesystem block size in bytes", gauge, []labelPair{{"ost", "lustrefs-OST0001"}}, 1.048576e+06, false},
-		{"lustre_lfsck_speed_limit", "Maximum operations per second LFSCK (Lustre filesystem verification) can run", gauge, []labelPair{{"ost", "lustrefs-OST0001"}}, 0, false},
-		{"lustre_shrink_requests_total", "Number of shrinks that have been requested", counter, []labelPair{{"ost", "lustrefs-OST0001"}}, 0, false},
-		{"lustre_exports_pending_total", "Total number of exports that have been marked pending", counter, []labelPair{{"ost", "lustrefs-OST0001"}}, 2.513043456e+10, false},
-		{"lustre_recalc_freed_total", "Number of locks that have been freed", counter, []labelPair{{"ost", "lustrefs-OST0001"}}, 0, false},
-		{"lustre_lock_contended_total", "Number of contended locks", counter, []labelPair{{"ost", "lustrefs-OST0001"}}, 32, false},
-		{"lustre_recovery_time_hard_seconds", "Maximum timeout 'recover_time_soft' can increment to for a single server", gauge, []labelPair{{
```

Comment:
No rush to merge this PR, so we'll take some time to figure out the right answer here.